export_proto_to_parquet/README.md

```diff
@@ -0,0 +1,59 @@
+# Temporal proto to parquet sample
+
+This is an example workflow to convert exported file from proto to parquet file. The workflow is an hourly schedule
+
+To use this code, make sure you have a [Temporal Cluster running](https://docs.temporal.io/docs/server/quick-install/) first.
+
+Create a virtual environment and activate it. On macOS and Linux, run these commands:
+
+```
+python3 -m venv env
+source env/bin/activate
+```
+
+On Windows, run these commands:
+
+```
+python -m venv env
+env\Scripts\activate
+```
+
+With the virtual environment configured, install the Temporal SDK:
+
+```
+python -m pip install temporalio
+python -m pip install pandas
+python -m pip install pyarrow
+python -m pip install boto3
+```
+
+
+Run the workflow:
+
+```bash
+python run_workflow.py
```

Comment:
```suggestion
python create_schedule.py
```

Can we change the filename to be a bit clearer on what's happening in the script?